webpack.config.js

```diff
@@ -81,7 +81,8 @@ module.exports = (env) => {
     },
     plugins: removeEmpty([
       new HtmlWebPackPlugin({
-        favicon: './client/public/assets/images/favicons/favicon.ico',
+        favicon:
+          './client/public/assets/images/favicons/wtt-halloween-favicon.png',
```

Comment:
Go ahead and use the `./client/src/assets/...` file path instead of this path that webpack builds. Webpack docs show to use the built path, but our config is different from the docs so not sure what the issue is atm.

Also, I think you are missing the `wtt-halloween-favicon.png` file? Can you commit that as well?

Reply:
Okay, will go ahead and add it to `./client/src/assets/...`

yeah It was committed, but for some reason after push it's doesn't get added, I figured it has something something with the fact that the public folder itself is not on remote. not sure what's going on but check it out. @jazhen

Reply:
As you said the public folder is gitignored so it wouldn't be committed. Thanks for bringing this issue up since it was going to affect all future favicon changes. Best to take care of it now.

Reply:
Thank you ! @jazhen 
made the changes now